templates/partner/intel.html

```diff
@@ -0,0 +1,645 @@
+          </p>
+        </div>
+        <div class="p-equal-height-row__item">
+          <p>
+            <a href="/ceph" aria-label="Learn more about Ceph">Learn more&nbsp;&rsaquo;</a>
+          </p>
+        </div>
+      </div>
+      <div class="p-equal-height-row__col">
+        <div class="p-equal-height-row__item ">
+          <div class="p-image-container--3-2-on-small p-image-container--square-on-medium p-image-container is-highlighted">
+            {{ image(url="https://assets.ubuntu.com/v1/f2fae2a2-kubeflow.png",
+                        alt="Kubeflow",
+                        width="852",
+                        height="480",
+                        hi_def=True,
+                        attrs={"class": "p-image-container__image"},
+                        loading="lazy") | safe
+            }}
+          </div>
+        </div>
+        <div class="p-equal-height-row__item">
+          <hr class="p-rule--muted u-hide--medium" />
+          <p>
+            Easily develop and deploy your machine learning models using Charmed Kubeflow, on any infrastructure including support for Intel GPUs.
+          </p>
+        </div>
+        <div class="p-equal-height-row__item">
+          <p>
+            <a href="/ai/what-is-kubeflow"
+               aria-label="Learn more about Charmed Kubeflow">Learn more&nbsp;&rsaquo;</a>
+          </p>
+        </div>
+      </div>
+      <div class="p-equal-height-row__col">
+        <div class="p-equal-height-row__item">
+          <div class="p-image-container--3-2-on-small p-image-container--square-on-medium p-image-container is-highlighted">
+            {{ image(url="https://assets.ubuntu.com/v1/661822e8-canonical.png",
+                        alt="Canonical",
+                        width="852",
+                        height="480",
+                        hi_def=True,
+                        attrs={"class": "p-image-container__image"},
+                        loading="lazy") | safe
+            }}
+          </div>
+        </div>
+        <div class="p-equal-height-row__item">
+          <hr class="p-rule--muted u-hide--medium" />
+          <p>
+            The Canonical Data Science Stack is an out-of-the-box solution to develop and optimize models, designed to run on any Intel workstation without spending time on tooling setup.
+          </p>
+        </div>
+        <div class="p-equal-height-row__item">
+          <p>
+            <a href="/ai/data-science"
+               aria-label="Learn more about Canonical data-science">Learn more&nbsp;&rsaquo;</a>
+          </p>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="row--50-50">
+      <hr class="p-rule" />
+      <div class="col">
+        <h2>Explore Ubuntu-certified Intel Hardware backed by Ubuntu Pro</h2>
+      </div>
+      <div class="col">
+        <div class="p-section--shallow">
```

Comment:
Similar to the previous comments, can you remove the shallow section on `p` and apply it to the entire row instead? This adheres to design requirements better as well